cluster/cluster.go

```diff
@@ -49,6 +53,10 @@ type SegConfig struct {
 	DbID      int
 	ContentID int
 	Role      string
+	PRole     string
```

Comment:
I would use the entire name here "PrimaryRole"

Reply:
Updated as "PreferredRole" to make it consistent with gp_segment_configuration output.